charts/sn-platform/values.yaml

```diff
@@ -987,6 +987,8 @@ bookkeeper:
     gcLoggingOptions: []
   custom: {}
   configData: {}
+    # For Pulsar 3.0, please do not enable Direct IO. Needs a fix https://github.com/apache/bookkeeper/pull/4041
+    # PULSAR_PREFIX_dbStorage_directIOEntryLogger: "true"
```

Comment:
Please change the default value to `false`

Reply:
@hangc0276 fixed in https://github.com/streamnative/charts/pull/1083/commits/462b4a7cbddc93ff662fec9262f71d8d85237333